src/api.ts

```diff
@@ -228,24 +228,22 @@ export class OpenSeaAPI {
   /**
    * Fetch list of assets from the API, returning the page of assets and the count of total assets
    * @param query Query to use for getting orders. A subset of parameters on the `OpenSeaAssetJSON` type is supported
-   * @param page Page number, defaults to 1. Can be overridden by
-   * `limit` and `offset` attributes from OpenSeaAssetQuery
    */
   public async getAssets(
-    query: OpenSeaAssetQuery = {},
-    page = 1
-  ): Promise<{ assets: OpenSeaAsset[]; estimatedCount: number }> {
+    query: OpenSeaAssetQuery = {}
+  ): Promise<{ assets: OpenSeaAsset[]; estimatedCount: number; next: string | undefined; previous: string | undefined }> {
```

Comment:
Could we change the `page` param to instead be a `cursor` param that takes in the cursor token from previous runs? That way users of `getAssets` will be able to paginate properly. Unless the `query` param already takes care of this?

Reply:
I suppose we could do that, but we'd need to add the cursor to the query object inside the function since that's the structure the API expects it in. I figured it would make sense to include all of the query params sent to the server in the query object passed to the `getAssets` method

You're a maintainer though so happy to refactor if that's what you'd prefer!

Reply:
This makes sense to me, was just clarifying. I think that should be fine to leave as-is for now. If this becomes problematic we can adjust as needed. 

@sunwrobert thoughts on the discussion above? Incidentally we are definitely overdue for testing here.

Reply:
Makes sense to keep the current changes. and yes we are overdue for testing 😄

Reply:
cool, approved as well